integration/tests/basic/test_rpc_calls.py

```diff
@@ -174,9 +175,55 @@ def test_eth_gas_price(self, param: tp.Union[str, None]):
             response["result"]
         ), f"Invalid current gas price `{response['result']}` in wei"
 
+    @pytest.mark.parametrize(
+        "blockhash, tag1, tag2", [
+            (True, None, None),
+            (True, Tag.EARLIEST, None),
+            (True, None, Tag.LATEST),
+        ]
+    )
+    def test_eth_get_logs_blockhash(self, event_caller_contract, blockhash, tag1, tag2):
```

Comment:
what about tests with blockhash + address or topic in parameters ? 
And some negative tests for all fields

Reply:
Added